carcara/src/checker/error.rs

```diff
@@ -225,6 +219,9 @@ pub enum CongruenceError {
 /// Errors relevant to the rules dealing with quantifiers.
 #[derive(Debug, Error)]
 pub enum QuantifierError {
+    #[error("lhs of argument equality is not a variable: '{0}'")]
+    ArgNotVar(Rc<Term>),
```

Comment:
Why was this error type added? It does not seem to be used anywhere

Reply:
It was used in an intermediate version. I had forgotten to delete it. It's removed now.